### PR TITLE
Feature/searchable select

### PR DESCRIPTION
**Pull request by gitsad**
* Added search functionality in select
* Extend props and updated docs

### CODE REVIEW

**Comment by irekrog** [comment]
Really good job 🌮  , but:

* I wonder why these files `docs/assets/main.js` and `docs/assets/search.js` are not minified right now? Are you sure you used `yarn typedoc` to generate these files? 
* Please add tests for `searchable` and `searchPattern` features
* I have noticed a few bugs
  - `disabled` prop doesn't work properly with `searchable` prop
  - open a dropdown, choose one of the options, open again the dropdown and you have now "No options". I think it should be work similar like react-select. Go to https://react-select.com/home and take a look for the example with "Searchable" prop
  - on a physical device (Android / iOS) keyboard is not closed after chosen option / pressed outside a dropdown
  - add `<Select searchable={true} ... />` in `Modal.tsx` example and after opened keyboard you will see a position of options list is wrongly computed
* Ideas (just ideas, not required in this PR 😄 )
  - maybe should we be add `onSearch`, `onFocus` and `onBlur` props callbacks?

File: src/components/select-input/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            });
+        }
+    };
+
+    return (
+        <TextInput
+            onChangeText={onChangeText}
+            onPressIn={onPressSelectControl}
+            placeholder={placeholderText}
+            style={styles.text}
```
Comment (irekrog) [inline src/components/select-input/index.tsx:50]: I think should be here also following props:
* `style={[styles.text, searchTextStyle]}`
* `placeholderTextColor={placeholderTextColor}`

Comment (gitsad) [reply src/components/select-input/index.tsx:50]: 🔥

**Review by koko57 — APPROVED** [review]

**Comment by gitsad** [comment]
Everything should be resolved now

**Review by irekrog — APPROVED** [review]

**Pull request merged**